### PR TITLE
Add Bloom 

### PR DESCRIPTION
**Pull request by Gornutz**
##### Name (to be shown on DefiLlama): 
Bloom

##### Twitter Link:
https://twitter.com/blueberryfdn

##### List of audit links if any:
https://docs.bloom.garden/audit

##### Website Link:
https://bloom.garden

##### Current TVL:
$1,46M

##### Treasury Addresses (if the protocol has treasury)
0xFdC004B6B92b45B224d37dc45dBA5cA82c1e08f2

##### Chain:
Ethereum

##### Short Description (to be shown on DefiLlama):
Permissionless real world asset yields for the DeFi economy

##### Category (full list at https://defillama.com/categories) *Please choose only one:
RWA

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
Chainlink

##### methodology (what is being counted as tvl, how is tvl being calculated):
Assets that are lent out on the Bloom protocol are counted as TVL. This includes the value of the underlying asset and the interest accrued.

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/blueberryfi

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/bloom exports TVL: 
        
 
 
    ethereum                  1.46 M
    
    total                    1.46 M

File: pull_request_template.md
Comment (g1nt0ki) [inline pull_request_template.md:1]: please revert changes to this file.

**Comment by llamatester** [comment]
The adapter at projects/bloom exports TVL: 
        
 
 
    ethereum                  1.46 M
    
    total                    1.46 M

File: pull_request_template.md
Comment (Gornutz) [reply pull_request_template.md:1]: reverted changes

**Pull request closed without merging**